tests/jenkins/pages/base.py

```diff
@@ -4,6 +4,9 @@
 
 class Base(Page):
 
+    def __init__(self, driver, base_url=None, timeout=20, **url_kwargs):
```

Comment:
If only we were able to use Python 3, [PEP3102](https://www.python.org/dev/peps/pep-3102/) would let us write this as:
```python
def __init__(self, timeout=20, *args, **kwargs):
```

One day! :-)